lib/extensions/string_extension.rb

```diff
@@ -39,6 +39,10 @@ def abbreviate(target_length = 6)
     acronym.upcase
   end
 
+  def to_legend
```

Comment:
**not a blocker** could we change this to either `to_slug`, `to_css_id`, `to_css_class` or something more generic as this method could possibly prove useful elsewhere.

Reply:
👍